src/OeUI/OemrUI.php

```diff
@@ -15,7 +15,7 @@
 
 // Special case where not setting up the header for a script, so using setupAssets function,
 //  which does not autoload anything. The actual header is set up in another script.
-Header::setupAssets(['jquery-ui', 'jquery-ui-base']);
+Header::setupAssets();
```

Comment:
In scripts that use this class, I am noting following js console errors:
![image](https://user-images.githubusercontent.com/278968/75642388-33f20880-5bf0-11ea-8006-f9deb9b273e9.png)